src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -51,993 +34,148 @@ def tearDown(self):
-    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
-    
-    # Request grant
-    config['grantRequests'][0]['cbsdId'] = cbsd_ids[0]
-    config['grantRequests'][0]['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
-    config['grantRequests'][0]['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
-    request = {'grantRequest': config['grantRequests']}
-
-    # Check grant response
-    response = self._sas.Grant(request)['grantResponse'][0]
-    self.assertEqual(response['cbsdId'], cbsd_ids[0])
-    self.assertIn(response['response']['responseCode'],config['expectedGrantResponseCodes'])
-    grant_id = response['grantId']
-    grant_expire_time = datetime.strptime(response['grantExpireTime'],
-                                          '%Y-%m-%dT%H:%M:%SZ')
-    del request, response
-   
-    # Relinquish grant1
-    request = {
-        'relinquishmentRequest': [{
-            'cbsdId': cbsd_ids[0],
-            'grantId': grant_id
-        }]
-    }
-    response = self._sas.Relinquishment(request)['relinquishmentResponse'][0]
-    # Check the relinquishment response
-    self.assertEqual(response['cbsdId'], cbsd_ids[0])
-    self.assertEqual(response['grantId'], grant_id)
-    self.assertIn(response['response']['responseCode'], config['expectedRelinquishResponseCodes'])
-    del request, response
-
-    # Load modified GWBL database
-    self._sas_admin.InjectDatabase_url(config['gwbl_db'])
-    
-    # Trigger daily activities
-    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
-
-    # Request grant
-    config['grantRequests'][0]['cbsdId'] = cbsd_ids[0]
-
-    config['grantRequests'][0]['operationParam']['operationFrequencyRange']['lowFrequency'] = 3660000000
-    config['grantRequests'][0]['operationParam']['operationFrequencyRange']['highFrequency'] = 3680000000
-    request = {'grantRequest': config['grantRequests']}
-
-    # Check grant response should be 400(INTERFERENCE)
+    config['grantRequest'][0]['operationParam']['operationFrequencyRange']['lowFrequency'] = freq_range[0]
+    config['grantRequest'][0]['operationParam']['operationFrequencyRange']['highFrequency'] = freq_range[1]
+    request = {'grantRequest': config['grantRequest']}
+    #Check grant response should be 400 (INTERFERENCE).
     response = self._sas.Grant(request)['grantResponse'][0]
     self.assertEqual(response['cbsdId'], cbsd_ids[0])
-    self.assertIn(response['response']['responseCode'],config['expectedGrantResponseCodes'])
-    grant_id = response['grantId']
-    grant_expire_time = datetime.strptime(response['grantExpireTime'],
-                                          '%Y-%m-%dT%H:%M:%SZ')
+    self.assertIn(response['response']['responseCode'], [400])
     del request, response
 
+    #Load modified exclusion zone database(change frequency of protected zone 'F2')
+    #Reading the fake database file.
+    with open(config['fake_fcc_db']['databaseFile'], 'r') as f:  
```

Comment:
The modification should be done in the config file, not hard-coded into the test case.

Reply:
Modified   the frequency range of Exclusion zone database.     (Loading Exclusion Zone Database file with Modified record).